apps/system/js/accessibility.js

```diff
@@ -284,11 +290,33 @@
     },
 
     /**
+     * Start a timeout that waits to display hints
+     * @memberof Accessibility.prototype
+     */
+    setHintsTimeout: function ar_setHintsTimeout(aDetails, aHints) {
+        clearTimeout(this.hintsTimer);
```

Comment:
Yes, I think forever is fine, but we need to make sure that we are quiet when the screen is off :)